packages/asset-buyer/src/utils/affiliate_fee_utils.ts

```diff
@@ -0,0 +1,29 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+
+import { SwapQuote, SwapQuoteInfo } from '../types';
+
+export const affiliateFeeUtils = {
+    getSwapQuoteWithAffiliateFee(quote: SwapQuote, feePercentage: number): SwapQuote {
```

Comment:
I still feel like these two `SwapQuotes` should be different types for clarity. Ie. `SwapQuote` vs. `SwapQuoteWithAffiliateFee`